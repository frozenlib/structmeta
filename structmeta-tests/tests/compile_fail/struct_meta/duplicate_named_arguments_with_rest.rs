#[structmeta_tests::attr_required_and_rest_named_value(value = 10, value = 10)]
fn my_func() {}

fn main() {}
