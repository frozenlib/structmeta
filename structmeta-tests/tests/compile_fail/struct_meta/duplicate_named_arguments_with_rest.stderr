error[E0433]: failed to resolve: could not find `attr_required_and_rest_named_value` in `structmeta_tests`
 --> $DIR/duplicate_named_arguments_with_rest.rs:1:21
  |
1 | #[structmeta_tests::attr_required_and_rest_named_value(value = 10, value = 10)]
  |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ could not find `attr_required_and_rest_named_value` in `structmeta_tests`
